doc/rtd/development/docs_layout.rst

```diff
@@ -0,0 +1,73 @@
+.. _docs_layout:
+
+Documentation directory layout
+******************************
+
+Cloud-init's documentation directory structure, with respect to the root
+directory:
+
+.. code-block:: text
+
+    /doc/
+        - examples/
+        - man/
+        - rtd/
+            - tutorial/
+            - howto/
+            - explanation/
+            - reference/
+
+            - development/
+            - static/
+                - css/
+                - js/
+                - *logos*
+            - *conf.py*
+            - *index.rst*
+            - *links.txt*
+        - rtd_html/
+        - sources/
+
+
+``examples/``
+=============
+
+
+``man/``
+========
+
+
+``rtd/``
+========
+
+This subdirectory is of most interest to anyone who wants to create or update
+either the content of the documentation, or the styling of it.
+
+* The content of the documentation is organised according to the `Diataxis`_
+  framework and can be found in the subdirectories: ``tutorial/``, ``howto/``,
+  ``explanation/``, and ``reference/``.
+
+* The ``development/`` subdirectory contains documentation for contributors.
+
+* ``static/`` contains content that relates to the styling of the documentation
+  in the form of custom CSS or javascript files found in ``css/`` and ``js/``
+  respectively. This is also where you can find the cloud-init logo.
+
+* ``conf.py`` contains Sphinx configuration commands.
+* ``index.rst`` is the front page of the documentation.
+* ``links.txt`` contains common (and reusable) links so that you do not need to
+  define the same URLs on every page and can use a more convenient shorthand
+  when referencing often-used links.
+
+``rtd_html/``
+=============
+
+When the documentation is built locally using ``tox -e doc``, the built pages
+can be found in this folder.
+
+``sources/``
```

Comment:
This is...uh...weird. It contains an example project of how to use some particular functionality (OVF) within cloud-init.  I'm guessing the original intention was to keep such demos here, but I didn't know this existed and not really sure it belongs here. We should probably document it as is until we move it though.